docs/changelog/1_0_a6.rst

```diff
@@ -0,0 +1,63 @@
+===========
+1.0 Alpha 6
+===========
+
+This changelog summarizes new features and breaking changes in
+EdgeDB 1.0 alpha 6 "Wolf".
+
+
+EdgeQL
+======
+
+* Introduce new enum definition syntax (:eql:gh:`#1843`).
+
+  Instead of using strings, the enums are now defined using identifiers:
+
+  .. code-block:: edgeql
+
+    CREATE SCALAR TYPE schema::Cardinality
+        EXTENDING enum<ONE, MANY>;
+
+* Rename built-in schema enums to use CamelCase names (:eql:gh:`#1843`).
+* Rename string handling functions to use ``start`` and ``end`` in
+  their name (:eql:gh:`#1752`).
+* Improve :ref:`UPDATE <ref_eql_statements_update>` functionality (
+  :eql:gh:`#1746`).
+* Forbid usage of DML in some special cases (:eql:gh:`#1726`).
+* Add tracking of DML inside function bodies (:eql:gh:`#1741`).
+* Allow functions with non-statement bodies (:eql:gh:`#1723`).
+* Allow the usage of :eql:op:`EXISTS` in constraints (:eql:gh:`#1750`).
+* Allow partial paths in object constraints (:eql:gh:`#1704`)
+* Allow trailing commas in collection types (:eql:gh:`#1749`).
+* Fix :eql:stmt:`INSERT ... UNLESS CONFLICT ... ELSE <INSERT>` when
+  combined with shape (:eql:gh:`#1743`).
+* Fix how :ref:`FOR <ref_eql_statements_for>` statement correlates
+  values (:eql:gh:`#1776`).
+* Fix handling of collections of newly created types in SDL (:eql:gh:`#1730`).
+* Fix handling of function definitions in SDL (:eql:gh:`#1649`).
+* Fix interactions of ``SET OF`` and ``OPTIONAL`` arguments (:eql:gh:`#1640`).
+* Implementation of more of the features of the new migration syntax
+  (`RFC 1000 <migrations_>`_).
+
+CLI
+===
```

Comment:
This title while is valid is still a bit confusing.

This is for the rust-cli/repl:
* Require instance name for most `edgedb server` commands.
* Add version check to show a warning when tools should be updated
* Add support for installing server using Docker (`edgdb server install --method=docker`)
* Add `edgedb self-upgrade` command

Reply:
I added the changes from `edgedb-cli` (sorry I missed that repo in my first attempt).

I am not fond of CLI title, but I'm also not sure what's a good simple name for this section.